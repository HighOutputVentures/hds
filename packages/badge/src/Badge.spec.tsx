describe("Badge", () => {
  it.todo("Should render badge");
  it.todo("Should render children");
  it.todo("Should render leftElement");
  it.todo("Should render rightElement");
  it.todo("Should render leftIcon");
  it.todo("Should render rightIcon");
});
