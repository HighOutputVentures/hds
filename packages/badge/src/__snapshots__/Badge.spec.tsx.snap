// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge Should match Badge snapshot 1`] = `
<div
  className="css-1jwgep6"
  data-testid="hds.badge.container"
  role="status"
>
  <p
    className="css-69eft0"
    data-testid="hds.badge-label"
  >
    Hello, Benjamen!😏
  </p>
</div>
`;
